jenkins/janitor.py

```diff
@@ -96,7 +96,7 @@ def clear_resources(project, col, resource):
     """
     err = 0
     for col, items in col.items():
-        if args.dryrun:
+        if args.dryrun is True:
```

Comment:
er, since it default to False, and type would still be str, isn't implicit truth is always going to be true? **(this seems to work, at least)